ambari-common/src/main/python/resource_management/core/files/killtree.sh

```diff
@@ -25,16 +25,16 @@ set -e
 killtree() {
     local _pid=$1
     local _sig=${2:--TERM}
-    ambari-sudo.sh kill -stop ${_pid} # needed to stop quickly forking parent from producing children between child killing and parent killing
+    /var/lib/amabari-agen/ambari-sudo.sh kill -stop ${_pid} # needed to stop quickly forking parent from producing children between child killing and parent killing
```

Comment:
It seems to be a Typo. I believe It was intended to be /var/lib/ambari-agent/ambari-sudo.sh